src/Tile.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react'
+import { Box, Span, theme } from '@truework/ui'
+import styled from 'styled-components'
+import { Field, FieldConfig, FieldProps } from 'formik'
+
+export type TileProps = {
+  id: string
+  icon?: React.ReactElement
+  label: string
+} & React.InputHTMLAttributes<HTMLInputElement>
+
+export type TileFieldProps = { name: string } & TileProps &
+  Pick<FieldConfig, 'validate'>
+
+const TileButton = styled.label(
+  ({ theme }) => `
+    width: 100%;
+    height: 100%;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    cursor: pointer;
+`
+)
+
+const TileContent = styled.div`
+  width: 100%;
+  height: 100%;
+  padding: ${theme.space.med} ${theme.space.sm};
+  top: 0px;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: ${theme.colors.body};
+  background-color: ${theme.colors.background};
+  border: 1px solid ${theme.colors.outline};
+  border-radius: ${theme.space.xxs};
+
+  transition-property: color, background-color, border-color;
+  transition-duration: ${theme.transitionDurations.fast};
+  transition-timing-function: ${theme.transitionTimingFunctions.ease};
+`
+
+const Input = styled.input`
+  margin: 0;
+  padding: 0;
+  appearance: none;
+
+  &:checked
+    ~ ${TileContent},
+    &:hover
+    ~ ${TileContent},
+    &:focus
+    ~ ${TileContent} {
+    border: 1px solid ${theme.colors.primary};
+    background-color: rgb(91, 99, 254, 0.1);
+  }
+
+  &:checked ~ ${TileContent} ${Span} {
```

Comment:
Notice how the color of the text doesn't transition when clicked? It's because the `transition-propert: color` is on `TileContent`, not the `Span`. So if you remove that here to update the color of the `TileContent` parent, it'll transition due to the cascade:

```suggestion
  &:checked ~ ${TileContent} {
```